uber/templates/registration/form.html

```diff
@@ -278,9 +281,12 @@
 <!-- TODO: integrate this with extra stuff -->
 <div class="form-group">
     <label class="col-sm-2 control-label">Badge Price</label>
-    <div class="col-sm-6">
+    <div class="checkbox col-sm-6">
         Base Price: $<input type="text" style="width:4em" name="overridden_price" value="{{ attendee.badge_cost|round(2) }}" />
-        <input type="checkbox" name="no_override" {% if attendee.overridden_price == None %} checked {% endif %}> Automatically recalculate badge price.
+        <label>
```

Comment:
I think this opening label tag needs to be moved down after the input? This bit of HTML is a bit odd but essentially it's an input box with a (logically but not programmatically) related checkbox next to it. So this would be:
```
<input type="checkbox" name="no_override" {% if attendee.overridden_price == None %} checked {% endif %}>
<label for="no_override"> Automatically recalculate badge price. </label>
```